app/distributions/api.py

```diff
@@ -132,8 +117,7 @@ def dataset(request: HttpRequest, dataset_id: int):
     Get the dataset with the given ID.
     """
     model = get_object_or_404(Dataset, id=dataset_id)
-    response = dataset_to_response(model)
-    return response
+    return model
```

Comment:
There's no need to explicitly call the DatasetSchema. Ninja does it when we tell it via decorator

Reply:
I see, nice! Didn't know it would be able to convert an integer to a string by itself.